SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -80,8 +80,10 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, QueuedIncomingRPCWaitTime(1.0f)
 	, QueuedIncomingRPCRetryTime(1.0f)
 	, QueuedOutgoingRPCRetryTime(1.0f)
-	, PositionUpdateFrequency(1.0f)
-	, PositionDistanceThreshold(100.0f) // 1m (100cm)
+	, PositionUpdateThresholdMinSeconds(1.0f)       // 1 second
+	, PositionUpdateThresholdMinDistance(100.0f) // 1m (100cm)
```

Comment:
I think this would be better if the units were in the variable names as you've done for the times.